### PR TITLE
Add recipe for atomicwrites 1.1.5

### PR DESCRIPTION
**Pull request by nicoddemus**
@untitaker as owner of the package, would you like to be included as maintainer of the conda package as well?

cc @tadeu

### CODE REVIEW

**Comment by untitaker** [comment]
No thanks. :)

On 11 April 2017 22:12:11 CEST, Bruno Oliveira <notifications@github.com> wrote:
>@untitaker as owner of the package, would you like to be included as
>maintainer of the conda package as well?
>
>cc @tadeu
>You can view, comment on, or merge this pull request online at:
>
>  https://github.com/conda-forge/staged-recipes/pull/2818
>
>-- Commit Summary --
>
>  * Add recipe for atomicwrites 1.1.5
>
>-- File Changes --
>
>    A recipes/atomicwrites/meta.yaml (49)
>
>-- Patch Links --
>
>https://github.com/conda-forge/staged-recipes/pull/2818.patch
>https://github.com/conda-forge/staged-recipes/pull/2818.diff
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/conda-forge/staged-recipes/pull/2818

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by nicoddemus** [comment]
I guess this is now ready to be merged then.

File: recipes/atomicwrites/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+test:
+  source_files:
+    - tests
+
+  imports:
+    - atomicwrites
+
+  commands:
+    - py.test tests
```
Comment (tadeu) [inline recipes/atomicwrites/meta.yaml]: 👍

**Comment by nicoddemus** [comment]
@conda-forge/core gentle ping. 😁

**Comment by nicoddemus** [comment]
Thanks @ocefpaf!

**Pull request merged**